content/goldcard-holders-faq/life-in-taiwan.md

```diff
@@ -73,3 +73,9 @@ These days, most foreign residents of Taiwan use the eGate exclusively to enter
  It costs 100NT and takes only a few minutes to produce. You can also
  [apply for one online](https://ncp.immigration.gov.tw/niaweb/entryExitForeignQuery) if
  you have a digital citizen certificate.
+
+## Can I start a company / employ myself?
```

Comment:
Under "at work" there is a question "Can I start a business with a Gold Card?". This probably needs to be merged somehow.

Reply:
Oh, I missed that one. Should we keep it keep the existing questions in "at-work" and link to the one in "life-in-taiwan". Or, move everything into one of them? I just felt that putting it under "Life In Taiwan" made more sense, whereas "At Work" sounds more like you already have a "job".

Reply:
I was wondering if "Start a Business" would deserve it own category under the "Life in Taiwan"? As you said, "At work" sounds like I have a job already? 

And we can definitely add more entrepreneurship questions in the "Start a Business" category

Reply:
( forgot to ping @fifieldt @pqvst )

Reply:
@erickhun Or maybe just rename "At Work" to "Work In Taiwan"? Then it would make more sense to put everything about work under that page. So basically, we have to main categories: "Life in Tawian" and "Work In Taiwan"

Reply:
@pqvst I like it.  to go even further... "Work/Business in Taiwan" ? no strong opinion about both, it's just I feel it's more explicit,  and since  we have many entrepreneurs folks. 

cc @fifieldt

Reply:
So, for now, I have moved my q/a into "at-work". I have also expanded the answers slightly to clearly distinguish between "setting up a company and employing yourself in that company" vs "being self-employed" (strictly, not the same thing).

For example, "self-employed" usually prefers to having a sole proprietorship rather than owning a company. My understanding is that this is fine, both abroad or, for example, locally in Taiwan if you do freelance work. Feel free to correct me if I'm wrong, but I believe that is correct.

I still think renaming "At Work" to something else would be good. "At Work" sounds like it's just about questions "in your workplace" or something like that.

Reply:
> I still think renaming "At Work" to something else would be good. "At Work" sounds like it's just about questions "in your workplace" or something like that.

Agree 👍 but let's open an issue and discuss about it for that one, I don't want to block that one